static/_redirects

```diff
@@ -65,18 +63,18 @@
 /docs/admin/static-pods/     /docs/tasks/administer-cluster/static-pod/ 301
 /docs/admin/sysctls/     /docs/tasks/administer-cluster/sysctl-cluster/ 301
 /docs/admin/resource-quota/     /docs/concepts/policy/resource-quotas/     301
-/docs/admin/upgrade-1-6/     /docs/tasks/administer-cluster/upgrade-downgrade/upgrade-1-6/ 301
+#/docs/admin/upgrade-1-6/     /docs/tasks/administer-cluster/upgrade-downgrade/upgrade-1-6/ 301
 
 /docs/api/     /docs/concepts/overview/kubernetes-api/ 301
 
 /docs/api-reference/labels-annotations-taints/     /docs/reference/labels-annotations-taints/ 301
-/docs/api-reference/v1.6/*     https://v1-6.docs.kubernetes.io/docs/reference/ 301
-/docs/api-reference/v1.7/*     https://v1-7.docs.kubernetes.io/docs/reference/ 301
-/docs/api-reference/v1.8/*     https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/:splat 301
-/docs/api-reference/v1.9/      https://v1-9.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/ 301
-/docs/api-reference/v1/definitions/    /docs/reference/generated/kubernetes-api/v1.10/ 301
-/docs/api-reference/v1/definitions.html /docs/reference/generated/kubernetes-api/v1.10/ 301
-/docs/api-reference/v1/operations/     /docs/reference/generated/kubernetes-api/v1.10/ 301
+#/docs/api-reference/v1.6/*     https://v1-6.docs.kubernetes.io/docs/reference/ 301
+#/docs/api-reference/v1.7/*     https://v1-7.docs.kubernetes.io/docs/reference/ 301
+#/docs/api-reference/v1.8/*     https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/:splat 301
+#/docs/api-reference/v1.9/      https://v1-9.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/ 301
+#/docs/api-reference/v1/definitions/    /docs/reference/generated/kubernetes-api/v1.10/ 301
+#/docs/api-reference/v1/definitions.html /docs/reference/generated/kubernetes-api/v1.10/ 301
+#/docs/api-reference/v1/operations/     /docs/reference/generated/kubernetes-api/v1.10/ 301
```

Comment:
If you're happy to remove these lines entirely, feel free.

Reply:
💯